packages/story-editor/src/components/publishModal/publishModal.js

```diff
@@ -109,6 +109,7 @@ function PublishModal({ isOpen, onPublish, onClose, publishButtonCopy }) {
       overlayStyles={{
         backgroundColor: theme.colors.inverted.interactiveBg.modalScrim,
       }}
+      id="modal-story-details-publish"
```

Comment:
Do we need this, can't we target this modal in some other way?

Reply:
This was how the previous publish modal is getting targeted by e2e so i was just mirroring that. can swap it to a testid.

Reply:
A testid would be more appropriate. Or even better targeting aria roles where possible.

Reply:
yeah i'll clean all of this up over here: https://github.com/GoogleForCreators/web-stories-wp/pull/10669